video/getting-started/use-participant-media/use-participant-media.m

```diff
@@ -0,0 +1,31 @@
+// Set the Participant Delegate to receive participant's events
+participant.delegate = self;
+
+// UIView where you want to render remote participant's video
+@property (nonatomic, weak) IBOutlet UIView *remoteView;
+
+#pragma mark - TVIParticipantDelegate
+
+- (void)participant:(nonnull TVIParticipant *)participant addedVideoTrack:(nonnull TVIVideoTrack *)videoTrack {
+    // Attach remoteView on video track to render video
+    [videoTrack attach:remoteView];
+}
+
+- (void)participant:(nonnull TVIParticipant *)participant removedVideoTrack:(nonnull TVIVideoTrack *)videoTrack {
+    // To stop rendering simply call detach
+    [videoTrack detach:remoteView];
```

Comment:
Needs to be either `_remoteView` or `self.remoteView` for the compiler to be happy.